tests/integration/negative_integration_test.sh.in

```diff
@@ -25,7 +25,7 @@ fi
 cd @CMAKE_CURRENT_BINARY_DIR@/@SIM_NAME@
 
 # We convert spikes to out.dat format
-reports=@ENABLE_SONATA_REPORTS@
+reports=@ENABLE_SONATA_REPORTS_TESTS@
 if [ "$reports" = "ON" ]
 then
   data=$(@H5DUMP_EXECUTABLE@ -d /spikes/timestamps -d /spikes/node_ids -y -O spikes | tr "," "\n")
```

Comment:
I think there is an issue with the datasets' names that fails the CI. It should be I believe:
```suggestion
  data=$(@H5DUMP_EXECUTABLE@ -d /spikes/All/timestamps -d /spikes/All/node_ids -y -O spikes | tr "," "\n")
  ```

Reply:
Yes! we are aware and working on it, this is the first issue that after fixing it we see new ones :D